xsd/netex_framework/netex_reusableComponents/netex_trainElement_version.xsd

```diff
@@ -597,31 +684,12 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Description of TRAIN COMPONENT.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:group ref="TrainElementViewGroup">
+			<xsd:group ref="TrainElementTypeViewGroup">
 				<xsd:annotation>
 					<xsd:documentation>Elements for a TRAIN ELEMENT derived.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:group>
 		</xsd:sequence>
 	</xsd:group>
 	<!-- ======================================================================= -->
-	<xsd:group name="TrainElementViewGroup">
```

Comment:
Is this removal intended?

Reply:
I think yes, at it is new TrainElementTypeView. Something which SBB opposes, but we will do this in a different issue https://github.com/NeTEx-CEN/NeTEx/issues/703